uni/assets/text/TermsAndConditions.md

```diff
@@ -52,30 +59,19 @@ Unless specifically permitted by you, your use of the Mobile Application does no
 license to link, reproduce, adapt, modify, publish or distribute the Content created by you or
 stored in your user account, for evaluating the effectiveness of existing product features, planning
 new features, or measuring size and characteristics about specific end-users, or devices with a data
-broker, for commercial, marketing or any similar purpose. Unless specifically permitted by you, your
+broker, for commercial, marketing or any similar purpose. 
+Unless specifically permitted by you, your
 use of the Mobile Application does not grant us the
 license to use analytics tools, advertising networks, third-party SDKs, or other external vendors
 whose code we have added to the Uni Mobile Application, to collect information such as user ID,
 device ID, or profiles with Third-Party Data for the purpose of accessing necessary data to
 implement
-the Services we provide. We do not have the license to sell or share sensitive data accessed trough
+the Services we provide. As of the last update, we anonymously track page visits and widget clicks, which is shared with our self-hosted instance of [Plausible Analytics](https://plausible.niaefeup.pt/).
+We do not have the license to sell or share sensitive data accessed trough
 API' permissions with the
 finality of facilitating sales.
```

Comment:
This would be more compact: `sensitive data collected with the purpose of facilitating sales`.